tests/test_main.py

```diff
@@ -0,0 +1,109 @@
+"""Unit tests for base_model.py"""
+
+from unittest.mock import Mock, patch
+
+import pytest
+from langchain_core.messages import AIMessage, HumanMessage
+from langgraph.graph import START
+
+from indigobot.__main__ import State, call_model, rag_chain, workflow
+
+
+@pytest.fixture
+def test_state():
+    """Test state fixture"""
+    return {
+        "input": "test question",
+        "chat_history": [],
+        "context": "test context",
+        "answer": "",
+    }
+
+
+@pytest.fixture
+def mock_rag_chain():
+    """Mock RAG chain fixture"""
+    with patch("langchain_app.base_model.rag_chain") as mock:
```

Comment:
Test fail at this line with follow error message:

name = 'langchain_app', import_ = <function _gcd_import at 0x7c18209d00e0>
>   ???
E   ModuleNotFoundError: No module named 'langchain_app'

<frozen importlib._bootstrap>:1324: ModuleNotFoundError
======== short test summary info =============
ERROR tests/test_main.py::test_call_model - ModuleNotFoundError: No module named 'langchain_app'

Reply:
Whoops. This one should be fixed now.